### PR TITLE
Liquidity pool transactions table

### PR DESCRIPTION
**Pull request by quietbits**
- `SortableTable` component (we can make it more generic later if needed)
- Added `poolTransactions` store and actions

image

### CODE REVIEW

File: src/pages/PoolsOverview.tsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { Layout } from "@stellar/design-system";
+import { PoolTransactionsList } from "components/PoolTransactionsList";
 
 export const PoolsOverview = () => (
-  <Layout.Inset>Pools overview page</Layout.Inset>
+  <Layout.Inset>
+    Pools overview page
+    <PoolTransactionsList poolId="b26c0d6545349ad7f44ba758b7c705459537201583f2e524635be04aff84bc69" />
+  </Layout.Inset>
```
Comment (quietbits) [inline src/pages/PoolsOverview.tsx:8]: Will remove this example before merging.

File: src/components/PoolTransactionsList.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      label: "Transaction",
+      sortBy: true,
+    },
+    // TODO: using shares for now. We probably need to fetch current prices to
+    // calculate total value.
+    {
+      id: "shares",
+      label: "Shares",
+      sortBy: true,
+    },
```
Comment (quietbits) [inline src/components/PoolTransactionsList.tsx:36]: Keeping it simple for now. Should be pretty straightforward to update to get total value.

File: src/helpers/fetchPoolTransactions.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+export const fetchPoolTransactions = async ({
+  server,
+  poolId,
+}: FetchPoolTransactionsProps): Promise<LiquidityPoolTransaction[]> => {
+  const operationsResponse = await server
+    .operations()
+    .forLiquidityPool(poolId)
+    .order("desc")
+    .call();
```
Comment (quietbits) [inline src/helpers/fetchPoolTransactions.ts:19]: Getting only LP deposit and withdraw operations, no trades yet.

File: src/helpers/getErrorString.ts [hunk 4]
```diff
@@ -0,0 +1,169 @@
+import { Horizon } from "stellar-sdk";
```
Comment (quietbits) [inline src/helpers/getErrorString.ts:1]: Copied this file from AV. We use this a lot, maybe it's worth adding to frontend helpers.

**Review by piyalbasu — APPROVED** [review]

**Pull request merged**